pipelines/auth/pip.yaml

```diff
@@ -0,0 +1,53 @@
+name: Configure pip to fetch Chainguard Libraries dependencies.
```

Comment:
Do we need separate pipelines for pip and uv, or could we just configure them both in the same pipeline?

Reply:
We cam throw them in the same pipeline